src/main/drivers/accgyro/accgyro_spi_icm42605.c

```diff
@@ -107,8 +107,6 @@ uint8_t icm42605SpiDetect(const busDevice_t *bus)
 {
     icm42605SpiInit(bus);
 
-    spiSetDivisor(bus->busdev_u.spi.instance, SPI_CLOCK_INITIALIZATION); //low speed
```

Comment:
Same question as above.

Reply:
Nothing in the data sheet says it is, but not being able to confirm that I'll revert it.